examples/rust/tcp-push-pop.rs

```diff
@@ -28,34 +29,36 @@ pub const SOCK_STREAM: i32 = libc::SOCK_STREAM;
 // Constants
 //======================================================================================================================
 
-const BUFSIZE_BYTES: usize = 64;
-const FILL_CHAR: u8 = 0x65;
-const ITERATIONS: usize = 10;
+const BUF_SIZE_BYTES: usize = 64;
+const ITERATIONS: usize = u8::MAX as usize;
 const TIMEOUT_SECONDS: Duration = Duration::from_secs(256);
 
-fn mksga(libos: &mut LibOS, size: usize, value: u8) -> Result<demi_sgarray_t> {
-    let sga: demi_sgarray_t = match libos.sgaalloc(size) {
+fn mksga(libos: &mut LibOS, value: u8) -> Result<demi_sgarray_t> {
+    let sga: demi_sgarray_t = match libos.sgaalloc(BUF_SIZE_BYTES) {
         Ok(sga) => sga,
         Err(e) => anyhow::bail!("failed to allocate scatter-gather array: {:?}", e),
     };
 
+    // Create pointer for filling the array.
+
```

Comment:
This is probably a left over blank line, it can be deleted.

Reply:
Thanks!